packages/node/src/node.ts

```diff
@@ -338,10 +388,15 @@ export class Node {
     const isProtocolMessage = (msg: NodeTypes.NodeMessage) =>
       msg.type === NODE_EVENTS.PROTOCOL_MESSAGE_EVENT;
```

Comment:
wont this return true for a `NodeMessageWrappedFinMessage` as well? Ref:

```javascript
await this.messagingService.send(to, {
        data,
        from: fromXpub,
        type: NODE_EVENTS.PROTOCOL_MESSAGE_EVENT
      } as NodeMessageWrappedFinMessage);
    });
```

Reply:
Yeah it will - the `FinMessage` can be thought of as a very specific type of `ProtocolMessage` as its purpose is to conclude a protocol, but its definition is different from the data that gets included in a `ProtocolMessage`